05observationSystem.cpp

```diff
@@ -0,0 +1,110 @@
+/*
+The array stores the values of the terrain profile heights with a constant horizontal step. 
+Find areas (elevation points) that are invisible to the observer at the first point.
+*/
+#include <iostream>
+#include <random>
+#include <time.h>
+
+#define QUANTITY 10
+
+using namespace std;
+
+void fillHeightRandom (double heightArray[QUANTITY]);
+void printAllHeights(double heightArray[QUANTITY]);
+bool isNotVisible(double heightOfOblerver, int numberOfPoint, double heightArray[QUANTITY]);
+void printHeight(double heightArray[], int i);
+
+int main()
+{
+	double heightArray[QUANTITY] = { 8,0,1,3,8,8,7,2,9,4 };
+
+	fillHeightRandom(heightArray);
+	printAllHeights(heightArray);
+
+	cout << "Not visible heights are: " << endl;	
+	for (int i = 2; i < QUANTITY; i++)
+	{
+		if (isNotVisible(heightArray[0], i, heightArray))
+		{
+			printHeight(heightArray, i);
+		}
+	}
```

Comment:
Shall I create one function like `void printNotVisible()` instead of these lines?